metaflow/plugins/__init__.py

```diff
@@ -137,28 +139,30 @@ def get_plugin_cli():
 AWS_CLIENT_PROVIDERS = resolve_plugins("aws_client_provider")
 SECRETS_PROVIDERS = resolve_plugins("secrets_provider")
 
+from .argo.argo_events import ArgoEvent
```

Comment:
Need to provide the alias but this needs to go before the merge.

Reply:
correct - although this seems to work without any issues.